promptsource/templates/math_qa/templates.yaml

```diff
@@ -30,25 +54,54 @@ templates:
 
       {{options}}|||
 
-      {{correct}}'
+      {{answer_choices[mapping[correct]]}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_3
-    reference: ''
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: choose_correct_og
+    reference: The template asks the model to choose the correct answer.
   82345ddb-f2e3-46dc-8283-531a3bd000e9: !Template
     answer_choices: null
     id: 82345ddb-f2e3-46dc-8283-531a3bd000e9
     jinja: "Given the problem:\n{{Problem}}\n===\nand the options:\n{{options}}\n\
       ===\nProvide a rationale for how you would solve the problem.  |||\n{{Rationale}}\
       \ \n"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_1
-    reference: ''
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: problem_options_give_rationale
+    reference: The template asks to generate the rationale
+  8c4c81cc-ca54-45fc-a69a-4b97a5f2b465: !Template
+    answer_choices: a ||| b ||| c ||| d
+    id: 8c4c81cc-ca54-45fc-a69a-4b97a5f2b465
+    jinja: "{% set mapping = {\"a\": 0, \"b\": 1, \"c\": 2, \"d\": 3} %}\nSolve the\
+      \ problem by choosing the correct answer: \n{{Problem}}\n\n{{options}}|||\n\
+      {{answer_choices[mapping[correct]]}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: pick_the_correct
+    reference: The template picks the correct the answer choice
+  a313a5f8-53cd-4b76-abb6-fea2ac4e9ef4: !Template
+    answer_choices: a ||| b ||| c ||| d
+    id: a313a5f8-53cd-4b76-abb6-fea2ac4e9ef4
+    jinja: "{% set mapping = {\"a\": 0, \"b\": 1, \"c\": 2, \"d\": 3} %}\nOne of the\
+      \ five choices are correctly answer the math problem. Can you choose the right\
+      \ one? \n\n{{options}}\n\nProblem: {{Problem}}\n|||\n{{answer_choices[mapping[correct]]}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: first_choice_then_problem
+    reference: First give the list of choices and then describe the problem
   a3c2ec72-4af5-42aa-9e8e-ef475fa7c039: !Template
     answer_choices: null
```

Comment:
The categories present in the dataset are the following:
```
>>> Counter(d["train"]["correct"])
Counter({'c': 6653, 'b': 6308, 'd': 6085, 'a': 6010, 'e': 4781})
>>> Counter(d["train"]["category"])
Counter({'general': 13273, 'physics': 7063, 'gain': 5120, 'geometry': 2117, 'other': 1814, 'probability': 450})
>>> Counter(d["test"]["category"])
Counter({'general': 1301, 'physics': 702, 'gain': 558, 'geometry': 200, 'other': 172, 'probability': 52})
>>> Counter(d["validation"]["category"])
Counter({'general': 2006, 'physics': 1056, 'gain': 743, 'geometry': 308, 'other': 288, 'probability': 74})
```

Could you update this field?